src/_forms.scss

```diff
@@ -0,0 +1,259 @@
+/**
+ * Wrapper class for all input fields
+ */
+.input, 
+.textarea, 
+.select {
+    margin: 0;
+    position: relative;
+    overflow: hidden;
+
+    /**
+    * Underline for input fields
+    */
+    &::after, 
+    &::before {
+        content: '';
+        width: 100%;
+        border-bottom: 2px solid #222;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+    }
+
+    /**
+    * After pseudo element for input underlines
+    * initially hidden outside by parent class
+    * overflow: hidden
+    * 1. Move the pseudo element to the left by 100%
+    * 2. Add animation effects to transform style
+    */
+    &::after {
+        margin-top: 1px;
+        border-bottom: 4px solid #51AFC1;
+        -webkit-transform: translate3d(-101%, 0, 0); /* 1 */
+	            transform: translate3d(-101%, 0, 0); /* 1 */
+        -webkit-transition: -webkit-transform 0.3s;  /* 2 */
+                transition:         transform 0.3s;  /* 2 */
+    }
+
+    /**
+    * After pseudo animated and comes into view
+    * when child input is focussed
+    */
+    &:focus-within::after {
+        -webkit-transform: translate3d(0, 0, 0);
+                transform: translate3d(0, 0, 0);
+    }
+}
+
+/**
+* General styling for the various inputs.
+* The input is placed inside a wrapper with a class
+* of the same name.
+*/
+.input input, 
+.textarea textarea, 
+.select select {
+    height: 2em;
+    width: 100%;
+    border: none;
+    padding: 0;
+    background: transparent;
+    font-size: 1em;
+    line-height: 1.5em;
+
+    &:focus {
+        outline: none;
+    }
+}
+
+/**
+* Textarea overrides
+*/
+.textarea textarea {
+    height: 4em;
+    line-height: 1.5em;
+    resize: none;
+}
+
+/**
+* Select input overrides. The padding is overriden.
+* Since the select box depends on each browser,
+* vendor prefixes are required to override some of the
+* styles.
+*/
+.select select {
+    height: 2em;
+    width: 100%;
+    font-size: 1em;
+    -webkit-padding-end: 0.5em 0;
+       -moz-padding-end: 0.5em;
+    -webkit-padding-start: 0;
+       -moz-padding-start: 0;
+    -webkit-user-select: none;
+       -moz-user-select: none;
+            user-select: none;
+    -webkit-appearance: button;
+       -moz-appearance: button;
+            appearance: button;
+
+    /**
+    * Select option padding overrides
+    * The padding is zeroed.
+    */
+    option {
+        -webkit-padding-end: 0;
+           -moz-padding-end: 0;
+        -webkit-padding-start: 0;
+           -moz-padding-start: 0;
+    }
+
+    /**
+    * Overrides for multi select boxes.
+    * Default select boxes are kept, the height
+    * is adjusted. The select color is updated.
+    * 1. In order to set the background of the multi select, 
+    *    a linear gradient was required
+    */
+    &[multiple] {
+        height: 4em;
+        overflow: scroll;
+
+        &:focus option:checked {
+            background: #51AFC1 linear-gradient(0deg, #51AFC1 0%, #51AFC1 100%); /* 1 */
```

Comment:
Can this colour be added to a variable? (we can change the value of the variable later but seems like it would be good to add in now)